File: src/resources/FormulaCode.java
public class FormulaCode {

	public static void main(String[] args){
        
		
	
	}
    public double convertOddsToDecimal(int number)
    {
        return number > 1 ? ((double)(number/100) + 1) : ((double)(100/number) + 1);
    }
    public double calculateProbability(double number)
    {
        return 1/number;
    }
    public boolean isArbitragePossible(double prob1, double prob2)
    {
        return ((prob1+prob2)<1)
    }
    public double calculateBet(int totalBet, double probability)
    {
        return totalBet*probability;
    }
}

File: src/resources/FormulaCode.js
export function convertOddsToDecimal(number){
    return number > 0 ? ((number/100.0) + 1) : ((100.0/number) + 1);
}

export function calculateProbability(number) {
    return 1.0/number;
}

export function isArbitragePossible(prob1, prob2) {
    return ((prob1+prob2)<1)
}

export function calculateBet(totalBet, probability) {
    return totalBet*probability;
}